discovery-infra/tests/base_test.py

```diff
@@ -604,7 +669,29 @@ def kube_api_context(self, kube_api_client):
         kube_api_context = KubeAPIContext(kube_api_client, clean_on_exit=global_variables.test_teardown)
 
         with kube_api_context:
+            v1 = CoreV1Api(kube_api_client)
+
+            try:
+                v1.create_namespace(body={
+                    "apiVersion": "v1",
+                    "kind": "Namespace",
+                    "metadata": {
+                        "name": global_variables.spoke_namespace,
+                        "labels": {
+                            "name": global_variables.spoke_namespace,
+                        },
+                    },
+                })
+            except K8sApiException as e:
+                if e.status == 409:
+                    pass
+                else:
+                    raise
```

Comment:
Use `kubernetes/dynamic/exceptions.py`
```suggestion
                if isinstance(e, ConflictError):
                    raise
```